### ISSUE TITLE
[Tab Tray Experiments][Accessibility] - The selector is not announced as selected

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/firefox-ios#26504]
## Steps to reproduce
1. Enable the Voice Over from iOS settings.
1. Open Firefox
1. Access tab tray.
1. Navigate from **Private** to **Tabs**, then to **Sync**.
1. Observe the behavior.

#### **Actual Result:**
The selector (*4tabs,private, sync*)  is not announced as selected by Voice Over.

#### **Expected Result:**
The selector is announced as selected by Voice Over.

### Device & build information
* Device: iPhone 14 Pro Max (iOS 16.2).
* Build version: v9000 (54788)

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/FXIOS-12174)
